.github/zip_and_upload_package.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -ex
+
+runner_os=$1
+tag_name=$2
+
+cd dist/
+
+if [ -f "./imagedephi.exe" ]; then
+    executable="imagedephi.exe"
+else
+    executable="imagedephi"
+fi
+
+chmod +x $executable
```

Comment:
I do see it being made executable here, but testing locally I had to change it manually.

Reply:
How did you test locally? I can try to reproduce on my end.

Reply:
I downloaded the linux artifact here https://github.com/DigitalSlideArchive/ImageDePHI/actions/runs/5578038631
extracted the file then tried to run the program from the terminal. I got a permission error.

Reply:
Ah I see! Thanks for explaining.

Reply:
Just to close the loop on this. I tested again locally using https://github.com/nektos/act and confirmed the unzipped file is executable.